solutions/src/main/java/ru/sberbank/school/task06/CountMap.java

```diff
@@ -4,18 +4,18 @@
 
 /**
  * Интерфейс мапы, которая ведет подсчёт вхождений элемента в контейнер.
- * <p>
- * <p>
+ *
+ *
```

Comment:
Empty line should be followed by &lt;p&gt; tag on the next line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1619833732>*